File: Alvas.Audio/AudioExVB/PositionEventArgs.vb
Imports System
Imports System.Text

Namespace AudioExCS
	Public Class PositionEventArgs
		Inherits EventArgs
		Public Sub New(ByVal position As Long)
			_position = position
		End Sub

		Private _position As Long
		Public ReadOnly Property Position() As Long
			Get
				Return _position
			End Get
		End Property

	End Class
End Namespace

File: Alvas.Audio/AudioExCS/PositionEventArgs.cs
using System;
using System.Text;

namespace AudioExCS
{
    public class PositionEventArgs : EventArgs
    {
        public PositionEventArgs(long position)
        {
            _position = position;
        }

        private long _position;
        public long Position
        {
            get { return _position; }
        }

    }
}
